source/hassio/haos_common_tasks.markdown

```diff
@@ -0,0 +1,140 @@
+---
+title: "Home Assistant OS Common Tasks"
+description: "Guides for common tasks when using Home Assistant OS"
+---
+
+
+## Configuring access to files
+
+One of the first things to take care of after installing Home Assistant OS is to provide yourself access to files. There are several add-ons commonly used for this, and most users employ a mix of a handful of add-ons. This section will describe some common choices.
+
+---
+
+### Installing and using the Samba add-on
+
+Your Home Assistant OS install has default directories which are mapped to the add-ons so that they can be accessed by the services a particular add-on might provide. The Samba add-on exposes all of these as smb shares which can then easily be accessed from another computer. You can also edit files using the editor of your preference from your client computer.
+
+The following directories are made available:
+
+- addons
+- backup
+- config
+- media
+- share
+- ssl
+
+On the host system these exist on the `/data` partition at `/mnt/data/supervisor/`.
+Configuring the add-on is straight-forward. You only need to set a user and password of your choice on the configuration page, save, and then start the add-on. The add-on will not start without setting a password! The other options can be left as-is for the majority of users.
+
+`interface:` - By default, this will use the currently configured network interface and does not need to be changed.
+
+`allow_hosts:` - This list of allowed networks covers the majority of home networks and does not usually need to be changed.
+
+`veto_files:` - These are files (mostly generated by OS X) which will not be passed into share directories
+
+- ._*
+- .DS_Store
+- Thumbs.db
+- icon?
+- .Trashes
+
+One issue many Windows 10 users run in to is that older versions of SMB are disabled by default due to security issues. This prevents users from browsing the network from file explorer in a way which might be expected. While this *could* be resolved by enabling SMBv1 on Windows, it is not advisable. Instead, you can enter the ip address or hostname of your Home Assistant instance in File Explorer's address bar with two backslashes, as shown in the example screenshot.
```

Comment:
Let's not mention an option and then tell users not to do it. Let's just not mention it at all.